### PR TITLE
qmp_command: update check method of "query-name","query-uuid"and "human-monitor-command"

### PR DESCRIPTION
**Pull request by jingzhao84**
qmp_command: update check method of "query-name","query-uuid"and "human-monitor-command"
ID: 1346721

Signed-off-by: jinzhao <jinzhao@redhat.com>

### CODE REVIEW

**Comment by jingzhao84** [comment]
(1/3) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-name: PASS (403.23 s)
 (2/3) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-uuid: PASS (42.85 s)
 (3/3) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.qmp_command.qmp_human-monitor-command: PASS (43.1

File: qemu/tests/cfg/qmp_command.cfg [hunk 1]
```diff
@@ -31,12 +31,11 @@
             cmd_return_value = "{u'status': u'paused', u'singlestep': False, u'running': False}"
         - qmp_query-name:
             qmp_cmd = "query-name"
-            post_cmd = "info name"
-            cmd_result_check = m_in_q
+            cmd_result_check = contain 
         - qmp_query-uuid:
+            uuid = 990ea161-6b67-47b2-b803-19fb01d30d12
```
Comment (yanan-fu) [inline qemu/tests/cfg/qmp_command.cfg]: Any strategy for using this value. or just a random value ?

**Comment by yanan-fu** [comment]
@jingzhao84  dirty branch , please check again, and add correct commit.

File: qemu/tests/cfg/qmp_command.cfg (hunk 1 above)
Comment (jingzhao84) [reply qemu/tests/cfg/qmp_command.cfg]: uuid just a random value for verify the result of "query-uuid"

File: qemu/tests/qmp_command.py [hunk 2]
```diff
@@ -266,7 +266,14 @@ def send_cmd(cmd):
         txt = "Verify that qmp command '%s' works as designed." % qmp_cmd
         logging.info(txt)
         if result_check == "equal" or result_check == "contain":
-            check_result(output, cmd_return_value, exception_list)
+            if qmp_cmd == "query-name":
+                vm_name = params["main_vm"]
+                check_result(output, vm_name, exception_list)
+            elif qmp_cmd == "query-uuid":
+                uuid_input = params.get("uuid")
```
Comment (ybduan) [inline qemu/tests/qmp_command.py]: I think params["uuid"] is better.

Comment (jingzhao84) [reply qemu/tests/qmp_command.py]: updated, thanks

**Comment by yanan-fu** [comment]
LGTM
Acked-by: Yanan Fu <yfu@redhat.com>

**Comment by ybduan** [comment]
LGTM
Acked-by: Yanbin Duan yduan@redhat.com

**Comment by luckyh** [comment]
@CongLi hi, would you please help review this pr? Thanks!

**Comment by jingzhao84** [comment]
@hereischen  could you help to review this pr? thanks

File: qemu/tests/cfg/qmp_command.cfg [hunk 3]
```diff
@@ -147,8 +146,8 @@
             negative_test = yes
         - qmp_human-monitor-command:
             qmp_cmd = human-monitor-command command-line=info kvm
-            post_cmd = info kvm
-            cmd_result_check = m_in_q
+            cmd_result_check = contain
+            cmd_return_value = "kvm support: enabled"
```
Comment (CongLi) [inline qemu/tests/cfg/qmp_command.cfg]: how about qemu 1.5.3 ? is it same value?

Comment (jingzhao84) [reply qemu/tests/cfg/qmp_command.cfg]: yes, the same value, please check the following result
{"QMP": {"version": {"qemu": {"micro": 3, "minor": 5, "major": 1}, "package": " (qemu-kvm-1.5.3-141.el7_4.2)"}, "capabilities": []}}
{"execute":"qmp_capabilities"}
{"return": {}}
{"execute":"human-monitor-command","arguments":{"command-line":"info kvm"}}
{"return": "kvm support: enabled\r\n"}

Comment (jingzhao84) [reply qemu/tests/cfg/qmp_command.cfg]: @CongLi  yes, the same value, please check the following executed result

{"QMP": {"version": {"qemu": {"micro": 3, "minor": 5, "major": 1}, "package": " (qemu-kvm-1.5.3-141.el7_4.2)"}, "capabilities": []}}
{"execute":"qmp_capabilities"}
{"return": {}}
{"execute":"human-monitor-command","arguments":{"command-line":"info kvm"}}
{"return": "kvm support: enabled\r\n"}

**Comment by CongLi** [comment]
ack.

**Comment by CongLi** [comment]
Thanks all.

**Pull request merged**